### ISSUE TITLE
Check token Payment Listen

### ISSUE THREAD

**Issue by ohari5336** [storj/storj#6642]
> /api/v0/payments/wallet/payments-with-confirmations 

### Response (on localhost):-
```
console service: storjscan service: storjscan client: tokens endpoint: tokens service: HeadersDB: storjscandb: violates constraint \"block_headers_pkey\": ERROR: duplicate key value violates unique constraint \"block_headers_pkey\" (SQLSTATE 23505)
```

**Comment by ohari5336** [comment]
I faced this issue when I ran a private geth Blockchain and deployed basic erc20 token.

After this I start storjscan with this and link with satellite via endpoint and api key, secret

After this setup I made a token transfer on storj billing address its detected transaction and shown in response. Now after saving these transactions in scan db then show this error on all new api calls.

When I delete this scan TRX data from db then api gives response first time then again giving error. 

Also TRX data after detection by scan and 20+ confirmation show msg in ui transaction successfully but in satellite db not saving and user account plan not upgraded
